core/templates/dev/head/css/oppia.css

```diff
@@ -280,13 +331,23 @@ body {
   }
 }
 
-h1, h2 {
+h2 {
   color: #000;
+  font-size: 1.5em;
+  margin: .83em 0;
+}
+
+h1 {
```

Comment:
Move h1 tag above h2 tag.

Reply:
Done

Reply:
Not done?

Reply:
The earlier commit was removed this is done in next commit PTAL!